<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkInForm</class>
 <widget class="QDialog" name="checkInForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check-in Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enregistrer Form&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="nameText"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="addText"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Nationality</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="nationality">
     <item>
      <property name="text">
       <string>India</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Belga</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Brasilero</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bangladesh</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bhutan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>China</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Myanmar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nepal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pakistan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sri Lanka</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Canadiense</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Francés</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Italiano</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mexicano</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Español</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="phone"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Occupation</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="occupation">
     <item>
      <property name="text">
       <string>Student</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Businessman</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Govt. Employee</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Corporate Employee</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Freelancer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Objet de la visite</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QComboBox" name="purpose">
     <item>
      <property name="text">
       <string>Tourism</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Academic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Business</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Official</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Others</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Les occupants totales</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QSpinBox" name="occupant">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Catégorie</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QComboBox" name="catList">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>:: select une ::</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>No Chambre</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QComboBox" name="roomList">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
